src/pages/[marketplace]/deals/new.tsx

```diff
@@ -52,22 +53,22 @@ const New: NextPageWithLayout = () => {
 
 			if (!credixPass) {
 				hide();
-				message.error({
-					content: intl.formatMessage({
+				notification.error({
+					message: intl.formatMessage({
 						defaultMessage: "No Credix Pass found for given public key",
 						description: "New deal: Credix Pass validation failed",
 					}),
 				});
+
 				return;
 			}
-		} catch {
+		} catch (error) {
 			hide();
-			message.error({
-				content: intl.formatMessage({
-					defaultMessage: "Failed to get Credix pass for given public key",
-					description: "New deal: Credix Pass request failed",
-				}),
+			notification.error({
+				message: "Failed to get Credix pass for given public key",
+				error,
```

Comment:
not sure if we need to pass this error here. if we reach this point then that means something went wrong with the request itself because otherwise we would have a credixpass or null. Not sure what this error might look like but I think it will be something not relevant for the end user. Maybe something like "Failed to fetch Credix pass" makes more sense here